tests/run/cpp_classes_def.pyx

```diff
@@ -249,3 +249,7 @@ def test_uncopyable_constructor_argument():
     cdef UncopyableConstructorArgument *c = new UncopyableConstructorArgument(
         unique_ptr[vector[int]](new vector[int]()))
     del c
+
+_WARNINGS="""
```

Comment:
Adding `_WARNINGS` doesn't do much by itself. If you also add `tag: warnings` then it checks that the warnings match what you given. If you're just aiming to ignore the warnings then it isn't needed.

Reply:
I think I wanted to be double safe by both removing the `werror` tag and adding `_WARNINGS`. Pragmatically, that warning comes from a test I didn't write, so I don't really know if matching that warning is important or not.
I can dive a bit into the code to see if that could be relevant to try to match that warning. If it's not relevant, I guess just removing the `werror` tag would be the right move.

Reply:
I think it depends on what the warning is...